### PR TITLE
18642 fix small screen preview and help links

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#18642` fix small screen preview and help links`](http://community.openproject.org/work_packages/18642)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-fix-18642-small-screen-details-pane.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-fix-18642-small-screen-details-pane.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/app/ui_components/wiki-toolbar-directive.js [hunk 1]
```diff
@@ -27,7 +27,7 @@
 //++
 
 module.exports = function() {
-  var HELP_LINK_HTML = '<a href="/help/wiki_syntax" class="icon icon-help" onclick="window.open(&quot;/help/wiki_syntax&quot;, &quot;&quot;, &quot;resizable=yes, location=no, width=600, height=640, menubar=no, status=no, scrollbars=yes&quot;); return false;">' + I18n.t('js.inplace.link_formatting_help') + '</a>',
+  var HELP_LINK_HTML = '<a href="/help/wiki_syntax" title="' + I18n.t('js.inplace.link_formatting_help') + '" class="icon icon-help" onclick="window.open(&quot;/help/wiki_syntax&quot;, &quot;&quot;, &quot;resizable=yes, location=no, width=600, height=640, menubar=no, status=no, scrollbars=yes&quot;); return false;">' + I18n.t('js.inplace.link_formatting_help') + '</a>',
```
Comment (0xF013) [inline frontend/app/ui_components/wiki-toolbar-directive.js:30]: @houndci sorry doggy, this piece of code is sacred

File: app/assets/stylesheets/content/_in_place_editing.sass [hunk 2]
```diff
@@ -10,8 +10,10 @@
     *
       cursor: wait!important
   &.preview
-    button, .help
+    button
```
Comment (myabc) [inline app/assets/stylesheets/content/_in_place_editing.sass:17]: @0xF013 I'd like to get rid of these custom overrides as much as possible. Can we not use `ng-hide` instead?

File: app/assets/stylesheets/content/_in_place_editing.sass [hunk 3]
```diff
@@ -169,3 +171,11 @@
       width: 30px
       .icon-context:before
         padding: 0 1px 1px 0
+@media(max-width: 1440px)
```
Comment (myabc) [inline app/assets/stylesheets/content/_in_place_editing.sass]: @0xF013 we have a predefined breakpoint that you can use here:

``` sass
@include breakpoint(xlarge)
  .inplace-editor .jstElements .help a

```

File: frontend/app/ui_components/wiki-toolbar-directive.js (hunk 1 above)
Comment (myabc) [inline frontend/app/ui_components/wiki-toolbar-directive.js:30]: @0xF013 this is why we need ES6 + multi-line/templated strings.

Comment (0xF013) [inline frontend/app/ui_components/wiki-toolbar-directive.js:30]: @myabc or, you know, not write such code

File: app/assets/stylesheets/content/_in_place_editing.sass (hunk 2 above)
Comment (0xF013) [inline app/assets/stylesheets/content/_in_place_editing.sass:17]: @myabc not here, it's some vanilla js plugin

File: app/assets/stylesheets/content/_in_place_editing.sass (hunk 3 above)
Comment (0xF013) [inline app/assets/stylesheets/content/_in_place_editing.sass]: @myabc  will do

Comment (0xF013) [inline app/assets/stylesheets/content/_in_place_editing.sass]: @myabc ok, those breakpoints are for min-width, and I need max-width

File: app/assets/stylesheets/content/_in_place_editing.sass [hunk 4]
```diff
@@ -160,3 +168,10 @@
       width: 30px
       .icon-context:before
         padding: 0 1px 1px 0
+@include breakpoint(xlarge)
+  .inplace-editor .jstElements .help a
+    width: auto!important
+    display: inline!important
+    height: auto!important
+    padding-top: auto!important
+    margin-bottom: auto!important
```
Comment (0xF013) [inline app/assets/stylesheets/content/_in_place_editing.sass:177]: @myabc it's kinda ugly because both styles apply

**Pull request merged**